src/components/MapView/Layers/AnalysisLayer/index.tsx

```diff
@@ -23,6 +23,10 @@ import {
   LayerDefinitions,
 } from '../../../../config/utils';
 
+function getRoundedData(data: number): string {
```

Comment:
Let's move this to a util function and make sure we use it in the table as well for homogeneity

Reply:
maybe we could also expose the number of decimals in this util. defaulting to 2